lib/Cleantalk/ApbctWP/Firewall/SFW.php

```diff
@@ -293,6 +297,23 @@ public function diePage($result)
 
             $status     = $result['status'] === 'PASS_SFW__BY_WHITELIST' ? '1' : '0';
             $cookie_val = md5($result['ip'] . $this->api_key) . $status;
+    
+            /**
+             * Message about IP status
+             */
```

Comment:
Maybe this looks nicer, don't you think?

`$message_ip_status = $result['status'] === 'PASS_SFW__BY_STATUS'
    ? __('IP in the common whitelist', 'cleantalk-spam-protect')
    : __('IP in the common blacklist', 'cleantalk-spam-protect');

$message_ip_status_color = $result['status'] === 'PASS_SFW__BY_STATUS'
    ? 'red'
    : 'green';`